### PR TITLE
Add signMessage method

### PR DESCRIPTION
**Pull request by Pessina**
This PR implements the signMessage method mentioned on this NEP: https://github.com/near/NEPs/blob/master/neps/nep-0413.md

### CODE REVIEW

**Comment by Pessina** [comment]
https://github.com/near/fast-auth-signer/assets/22358004/c6ae8a26-ea6e-4068-b709-14b51f931a1f

File: packages/near-fast-auth-signer/src/components/SignMessage/SignMesage.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    postMessageToParent({
+      ok: false,
+    });
+  }, []);
+
+  return (
+    <ModalSignWrapper ref={signTransactionRef}>
+      <div className="modal-body">
+        <div className="modal-top">
+          <ModalIconSvg />
```
Comment (gabehamilton) [inline packages/near-fast-auth-signer/src/components/SignMessage/SignMesage.tsx:160]: Can we switch to the Near icon instead of the `?` icon?

**Review by gabehamilton — COMMENTED** [review]
LGTM

**Review by esaminu — APPROVED** [review]
LGTM! Great work abstracting out the iframe requests. I also tested multi chain locally with the refactor and it works great ✅

**Pull request merged**